src/ILICheck.Web/GeoJsonHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using DotSpatial.Projections;
+using ILICheck.Web.XtfLog;
+using NetTopologySuite.Features;
+using NetTopologySuite.Geometries;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ILICheck.Web
+{
+    /// <summary>
+    /// Provides helper methods for the GeoJSON (RFC 7946) format.
+    /// </summary>
+    public static class GeoJsonHelper
+    {
+        private static readonly ProjectionInfo lv95 = ProjectionInfo.FromEpsgCode(2056);
+
+        /// <summary>
+        /// Converts XTF log entries to a GeoJSON feature collection.
+        /// </summary>
+        /// <param name="logResult">The XTF log entries.</param>
+        /// <returns>A feature collection containing the log entries.</returns>
+        public static FeatureCollection CreateFeatureCollection(IEnumerable<LogError> logResult)
+        {
+            if (!AllCoordinatesAreLv95(logResult))
+            {
+                return null;
+            }
+
+            var features = logResult
+                    .Where(log => log.Geometry?.Coord != null)
+                    .Select(log => new Feature(ProjectLv95ToWgs84(log.Geometry.Coord), new AttributesTable(new KeyValuePair<string, object>[]
+                    {
+                        new ("type", log.Type),
+                        new ("message", log.Message),
+                        new ("objTag", log.ObjTag),
+                        new ("dataSource", log.DataSource),
+                        new ("line", log.Line),
+                        new ("techDetails", log.TechDetails),
+                    })));
+
+            var featureCollection = new FeatureCollection();
+            foreach (var feature in features)
+            {
+                featureCollection.Add(feature);
+            }
+
+            return featureCollection;
```

Comment:
Q: Kann für das Erstellen der `FeatureCollection` nicht der zweite Konstruktor mit dem Parameter `Collection<IFeature> features` benutzt werden? Dafür muss vermutlich `features` in eine Liste konvertiert werden weiter oben.

Reply:
Bei der `FeatureCollection` aus NetTopologySuite ist nur der leere Konstruktor public (und es gibt auch keine `AddRange` Methode).